src/Model/Command/ItemProperty.php

```diff
@@ -32,6 +32,17 @@ protected function setItemId(string $itemId): void
         $this->itemId = $itemId;
     }
 
+    protected function setProperties(array $properties): void
+    {
+        Assertion::keyNotExists(
+            $properties,
+            'item_id',
+            '$properties cannot contain key item_id - it is used as primary key to identify the item. Please use different key instead.'
```

Comment:
`Cannot update value of "item_id" property - it is used by Matej to identify the item and cannot be altered once created.` ?